README.md

```diff
@@ -67,7 +67,7 @@ Checkout the [Tutorials](https://move2kube.konveyor.io/tutorials) and [Documenta
 
 ## Development environment setup
 
-To browse code [![Open in VSCode](https://open.vscode.dev/badges/open-in-vscode.svg)](https://open.vscode.dev/konveyor/move2kube)
+To browse code  [![Open in VSCode](https://badgen.net/badge/icon/visualstudio?icon=visualstudio&label)](https://open.vscode.dev/konveyor/move2kube)
```

Comment:
Can you please change `https://badgen.net/badge/icon/visualstudio?icon=visualstudio&label` to `https://badgen.net/badge/icon/Visual%20Studio%20Code?icon=visualstudio&label`

Reply:
updated 👍